Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerCommandCollectionViewCell.swift

```diff
@@ -51,6 +51,7 @@ open class _ChatMessageComposerCommandCellView<ExtraData: ExtraDataTypes>: _View
 
         addSubview(textStackView)
         setupStack()
+        commandNameLabel.setContentCompressionResistancePriority(.streamRequire, for: .horizontal)
```

Comment:
Better if you set it to `.streamLow` so it doesn't mess with other `streamRequire` constraints.

Reply:
✅